addon/components/connect.js

```diff
@@ -37,9 +38,16 @@ export default (stateToComputed, dispatchToActions=() => ({})) => {
           });
         }
 
-        this.actions = Object.assign({},
-          this.actions, dispatchToActions.call(this, redux.dispatch.bind(redux))
-        );
+        if (typeof dispatchToActions === 'function') {
+          this.actions = Object.assign({},
+            this.actions, dispatchToActions.call(this, redux.dispatch.bind(redux))
+          );
+        }
+
+        if (typeof dispatchToActions === 'object') {
```

Comment:
That's a good point. There is reason to what seems to be an error, though.

redux's `bindActionCreators` uses `Object.keys` to iterate over the object passed in to it. If given an array of functions, it will work just fine, as `Object.keys([() => 'func'])` will simply return `["0"]`, thus making it possible to look up by index from the original array, and do its thing. 

So while no-one actually might do it, we can actually use an object *or* an array as the value of `dispatchToActions`. It also gives us parity with the `react-redux` `connect` HOC (it only checks for `typeof mapDispatchToProps === 'object'`), so we get some docs for free 👍